### PR TITLE
Added eslint and prettier

### PR DESCRIPTION
**Pull request by Samridhi-98**
# Related Issue
- Fixes : #68 

# Proposed Changes
- Add .eslintrc and .eslintignore 
- Add .prettierrc and .prettierignore

# Checklist
- [x] Add .eslintrc.json and .eslintignore and add the linting rules
- [x] Add .prettierrc and .prettierignore and add the linting rules
- [x] Add eslint and prettier as devDependencies in package.json
- [x] Add linting scripts to package.json

### CODE REVIEW

**Comment by Samridhi-98** [comment]
@ShauryaAg  @paramsiddharth  Please review my PR.

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "tabWidth": 2,
```
Comment (ShauryaAg) [inline .prettierrc:2]: Please set `tabWidth` to 4

**Review by ShauryaAg — CHANGES_REQUESTED** [review]

File: .prettierrc (hunk 1 above)
Comment (Samridhi-98) [reply .prettierrc]: Done!
Anything else to improve?

**Comment by ShauryaAg** [comment]
LGTM!

**Review by ShauryaAg — APPROVED** [review]

**Pull request merged**